shared/chat/conversation/messages/wrapper/index.js

```diff
@@ -1,5 +1,530 @@
 // @flow
-import WrapperMessage from './wrapper-message'
-import WrapperAuthor from './wrapper-author'
+import * as Kb from '../../../../common-adapters'
+import * as React from 'react'
+import * as Styles from '../../../../styles'
+import * as Types from '../../../../constants/types/chat2'
+import * as Constants from '../../../../constants/chat2'
+import SystemAddedToTeam from '../system-added-to-team/container'
```

Comment:
instead of MessageFactory which is a connect + a big switch statement, which then delegates to the wrapper 100% of the time, we just have wrapper, which knows how to render a message